resin/metadata.csv

```diff
@@ -0,0 +1,13 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+resin.connection_count,gauge,10,connection,,The current connection count for a particular connection pool,0,resin,connection count
```

Comment:
the aliases changes need to be reflected on this file too

Reply:
Done